build-system/server/app-utils.js

```diff
@@ -80,64 +82,81 @@ const isRtvMode = (serveMode) => {
   return /^\d{15}$/.test(serveMode);
 };
 
+const tsModule = Object.create(null);
+
+/**
+ * @param {string} path
+ * @return {*}
+ */
+function requireTsSync(path) {
+  if (!tsModule[path]) {
+    const outfile = `build/${path.replace(/\//g, '--')}.js`;
+    logLocalDev('Building', cyan(path), '...');
+    esbuild.buildSync({
+      entryPoints: [path],
+      format: 'cjs',
+      write: true,
+      outfile,
+    });
+    tsModule[path] = require(`${relative(
+      __dirname,
+      process.cwd()
+    )}/${outfile}`);
+    logLocalDev('Built', cyan(path));
```

Comment:
nit: When and how often does this get printed? If it's during server launch, I think the logging convention we use is to only print the post-build message.

Reply:
It occurs during the first usage, which is usually when the first AMP doc is served. I changed it. Let me know if that's not appropriate.